src/index.html

```diff
@@ -4,9 +4,359 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <section class="container">
+        <div class="container__item">
+          <form class="card" data-qa="card">
+            <img
+              class="card__photo"
+              src="images/imac.jpeg"
+              alt="Imac"
+            >
+            <p class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+            <p class="card__num">Код товара: 195434</p>
+            <div class="card__stars-line">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
```

Comment:
blank spasings are not required for single-line elements. Lets remove.

Reply:
Fixed